workers-site/index.js

```diff
@@ -1,10 +1,22 @@
-import { getAssetFromKV, defaultKeyModifier} from '@cloudflare/kv-asset-handler'
+import {
+  getAssetFromKV,
+  defaultRequestKeyModifier,
+} from '@cloudflare/kv-asset-handler'
 
 // do not set to true in production!
-const DEBUG = false
+const DEBUG = true
 
 addEventListener('fetch', event => {
-  event.respondWith(handleEvent(event))
+  try {
+    event.respondWith(handleEvent(event))
+  } catch (e) {
+    if (DEBUG) {
+    return event.respondWith(new Response(e.message || e.toString(), {
+      status: 404,
+    }))
+  }
```

Comment:
nit: indentation on these two lines i think; the next line is still inside this catch.

Reply:
I used prettier